group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java

```diff
@@ -599,13 +599,20 @@ public MemberState consumerGroupMemberState(
             .state();
     }
 
+
     public CoordinatorResult<ConsumerGroupHeartbeatResponseData, CoordinatorRecord> consumerGroupHeartbeat(
         ConsumerGroupHeartbeatRequestData request
+    ) {
+        return this.consumerGroupHeartbeat(ApiKeys.CONSUMER_GROUP_HEARTBEAT.latestVersion(), request);
+    }
+
+    public CoordinatorResult<ConsumerGroupHeartbeatResponseData, CoordinatorRecord> consumerGroupHeartbeat(
+        short apiVersion, ConsumerGroupHeartbeatRequestData request
```

Comment:
nit1: Let's put apiVersion as the second argument. This is usually what we do.
nit2: Let's put one argument per line to follow the style of the code in this file.